pkg/server/server.go

```diff
@@ -0,0 +1,74 @@
+package server
+
+import (
+	"fmt"
+	"github.com/labstack/echo/v4/middleware"
+	"net/http"
+	"time"
+
+	"github.com/labstack/echo/v4"
+)
+
+const basePhotosApiPath = "/v1/photos"
+
+type Server struct {
+	port       string
+	echoServer *echo.Echo
+}
+
+type status struct {
+	Status int `json:"status"`
+}
+
+func InitServer(port string) *Server {
+	// Echo instance
+	e := echo.New()
+	// Middleware
+	e.Use(middleware.Logger())
+
+	s := &Server{
+		port:       port,
+		echoServer: e,
+	}
+	s.AddEndpoints()
+	return s
+}
+
+func (server *Server) statusEndpoint(c echo.Context) error {
+	return c.JSON(http.StatusOK, &status{Status: http.StatusOK})
+}
+
+func getMarsRoverImagesForDate(earthDate string) string {
+	log := "Fetching Mars Rover images for " + earthDate
+	fmt.Println(log)
+	return log
+}
+
+func (server *Server) marsRoverImagesForEarthDateEndpoint(c echo.Context) error {
```

Comment:
This method gets the mars rover images for a date specified on the url path - `/v1/photos/2023-08-01`